.env

```diff
@@ -0,0 +1,8 @@
+# Environment variables declared in this file are automatically made available to Prisma.
```

Comment:
A suggestion would be to have this file be called `.env-sample` and have all other `.env` files be included in `.gitignore` just so someone doesn't include something they didn't mean to later on to the repo